.gitignore

```diff
@@ -0,0 +1,4 @@
+target/**
```

Comment:
Did you lose your local gitignore somwhere?;) Where did that folder come from "*
charts/data-space-connector/charts/**" ?